### PR TITLE
docs/flow: use consistent placeholders

### PR DESCRIPTION
**Pull request by rfratto**
Placeholders should be all-uppercase SNAKE_CASE with no other special characters denoting a placeholder (such as angle brackets).

### CODE REVIEW

**Review by mattdurham — APPROVED** [review]
Lgtm

File: docs/sources/flow/config-language/syntax.md [hunk 1]
```diff
@@ -35,9 +35,9 @@ The following example sets the `log_level` attribute to `"debug"`.
 log_level = "debug"
 ```
 
-All `<ATTRIBUTE_NAME>`s must be valid River [identifiers](#identifiers).
+All `ATTRIBUTE_NAME`s must be valid River [identifiers](#identifiers).
```
Comment (karengermond) [inline docs/sources/flow/config-language/syntax.md:38]: Plurals with code elements are awkward.  (Can't find the google dev style guideline at the moment). I think we should write around this if possible as I did here.

Comment (karengermond) [inline docs/sources/flow/config-language/syntax.md:38]:
```suggestion
The `ATTRIBUTE_NAME must be a valid River [identifier](#identifiers).
```

File: docs/sources/flow/config-language/syntax.md [hunk 2]
```diff
@@ -50,21 +50,21 @@ of arguments and nested unlabeled blocks.
 
 ```
 // Pattern for creating an unlabeled block:
```
Comment (karengermond) [inline docs/sources/flow/config-language/syntax.md:52]: Might want to make the comment into a heading ...  but do we go down to H4s?

File: docs/sources/flow/config-language/syntax.md [hunk 3]
```diff
@@ -50,21 +50,21 @@ of arguments and nested unlabeled blocks.
```
Comment (karengermond) [inline docs/sources/flow/config-language/syntax.md:50]: #### Pattern for creating an unlabeled block

File: docs/sources/flow/config-language/syntax.md [hunk 4]
```diff
@@ -50,21 +50,21 @@ of arguments and nested unlabeled blocks.
...
+BLOCK_NAME {
   // Block body can contain attributes and nested unlabeled blocks
-  <IDENTIFIER> = <EXPRESSION> // Attribute
+  IDENTIFIER = EXPRESSION // Attribute
 
-  <NESTED_BLOCK_NAME> {
+  NESTED_BLOCK_NAME {
     // Nested block body
   }
 }
```
Comment (karengermond) [inline docs/sources/flow/config-language/syntax.md:61]:
```suggestion
```

#### Pattern for creating a labeled block

```

File: docs/sources/flow/config-language/syntax.md [hunk 5]
```diff
@@ -50,21 +50,21 @@ of arguments and nested unlabeled blocks.
...
 
 // Pattern for creating a labeled block:
-<BLOCK NAME> "<BLOCK LABEL>" {
+BLOCK_NAME "BLOCK_LABEL" {
```
Comment (karengermond) [inline docs/sources/flow/config-language/syntax.md:63]:
```suggestion
```river
BLOCK_NAME "BLOCK_LABEL" {
```

**Review by karengermond — APPROVED** [review]
LGTM

**Pull request merged**